Utilities/File.cpp

```diff
@@ -1235,6 +1251,28 @@ fs::file::file(const std::string& path, bs_t<open_mode> mode)
 		{
 			return m_handle;
 		}
+
+		file_id get_id() override
+		{
+			file_id id{"windows_file"};
+			id.data.resize(sizeof(FILE_ID_INFO));
+
+			FILE_ID_INFO info;
+
+			if (!GetFileInformationByHandleEx(m_handle, FileIdInfo, &info, sizeof(info)))
```

Comment:
Windows' implementation is copied from MSVC's STL https://github.com/microsoft/STL/blob/d6f9987d7ec694b8599fe21e87e3653c36566223/stl/src/filesystem.cpp#L419